Relazione hcsr04

xn3cr0nx opened the pull request:
Nel procedimento mi piacerebbe andare a capo dopo i due punti in ogni parte delle prove, ma non so come si faccia senza lasciare una riga che viene brutto. Chi sa farlo ben venga

Commit:
Prima bozza di relazione

Commit:
Rename ScheletroRelazione.md to Bozz Relazione.md

Commit:
Update and rename Bozz Relazione.md to Bozza Relazione.md

lorisrossi commented:
Per andare a capo, puoi usare il tag html `<br>`
Ottimo lavoro, non ho guardato il codice visto che la relazione si spiega da sè

Commit:
Andato a capo

Commit:
Upload file misurazioni

Ho raggruppato in un unico file tutte le misurazioni fatte per le prove a diverse distanze

Commit:
Allegato file misure

Commit:
Rename Comprensivo_prove.csv to Misure_prove.csv

sebgiles commented:
Ottimo lavoro, per me si può procedere al merge

lorisrossi commented:
Prima del merge toglierei "Bozza" dal titolo, inoltre bisogna decidere se fare le prove con le pareti fonoassorbenti. Secondo me non sono necessarie. Se si vogliono fare, direi di aspettare di fare il merge finchè non si completa, se invece non si faranno anche io sono d'accordo per il merge

sebgiles commented:
sì sono d'accordo, finchè è una bozza non appartiene sul master 👍

sebgiles commented:
Si potrebbe aggiungere un minuscolo paragrafo in cui si elencano i risultati pratici utili a noi tipo:

angolo massimo rilevazione: XXX
field of view: XXX
tempo rubato alla CPU: XXX

sebgiles commented:
se vuoi proprio fare una cosa figa prova a plottare qualcosa

xn3cr0nx commented:
Si può fare tutto, da open office il plot ci ho provato ma ho davvero lasciato perdere. Magari con matlab buttandoci dentro i dati sarebbe più easy. Per la tabella si può fare, magari con un confronto con i datasheet appena ho un attimo

sebgiles commented:
non chiudo la fork in caso tu voglia fare altre modifiche, ma almeno su master ci metto questa roba che abbiamo adesso
